macros/edr/data_monitoring/data_monitors_configuration/get_buckets_configuration.sql

```diff
@@ -8,6 +8,18 @@
     {{ return(global_min_bucket_start) }}
 {% endmacro %}
 
+{# bucket_end represents the end of the bucket, so we need to add extra day to the timedelta #}
+{% macro get_global_min_bucket_end() %}
```

Comment:
I don't think this macro is in use, right?
If not remove it. 
If it is - I would make 'get_global_min_bucket_end(String=False)', returnd datetime by default and String if String=True